src/tracks/models/dto/track.dto.ts

```diff
@@ -1,17 +1,16 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { Exclude, Expose } from 'class-transformer';
-import { IsAlphanumeric, IsNotEmpty, IsOptional, MaxLength } from 'class-validator';
+import { IsAlphanumeric, IsNotEmpty, IsOptional, MaxLength, isAlphanumeric } from 'class-validator';
```

Comment:
Codacy found an issue: ['IsAlphanumeric' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=6064848)